src/khoj/main.py

```diff
@@ -126,6 +127,24 @@ def run(should_start_server=True):
     # Setup task scheduler
     poll_task_scheduler()
 
+    # Setup Background Scheduler
+    from django.conf import settings as django_settings
```

Comment:
Can this go to `settings.py`?

Reply:
We import the `django_settings` here to build the DB URL to pass to APScheduler for scheduled job persistence.